service/src/main/java/bio/terra/cbas/runsets/monitoring/SmartRunsPoller.java

```diff
@@ -47,12 +47,15 @@ public SmartRunsPoller(
     this.objectMapper = objectMapper;
   }
 
-  public RecordResponse updateOutputAttributes(Run run) throws Exception {
-    List<WorkflowOutputDefinition> outputDefinitionList =
-        objectMapper.readValue(run.runSet().method().outputDefinition(), new TypeReference<>() {});
+  public RecordResponse updateOutputAttributes(
+      Run run, List<WorkflowOutputDefinition> outputDefinitionList) throws Exception {
     Object outputs = cromwellService.getOutputs(run.engineId());
     RecordAttributes outputParamDef = OutputGenerator.buildOutputs(outputDefinitionList, outputs);
     RecordRequest request = new RecordRequest().attributes(outputParamDef);
+
+    logger.info(
+        "Updating output attributes for Record ID {} from Run {}", run.recordId(), run.engineId());
```

Comment:
This feels more like debug than info. And TBH I'd probably avoid putting confidential user values in logs at all, if we can avoid it.

Reply:
I can change this to debug level. As we discussed in standup, this is logging only the Record ID and Run ID hence I will leave it as it is.